components/src/dynamo/trtllm/tests/test_trtllm_video_diffusion.py

```
@@ -390,28 +390,37 @@ class TestVideoData:
 
     def test_url_only(self):
         """Test VideoData with URL only."""
-        data = VideoData(url="/tmp/video.mp4")
+        data = VideoData(response_format="mp4", url="/tmp/video.mp4")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/video.mp4"

(S108)